### ISSUE TITLE
Understanding for 1.4.10 redefines normative requirement for 1.4.4

### ISSUE THREAD

**Issue by patrickhlauke** [w3c/wcag#1839]
https://www.w3.org/WAI/WCAG21/Understanding/reflow.html#the-relation-of-reflow-to-the-success-criterion-1.4.4-resize-text states that

> It is not required to achieve 200% text enlargement at every breakpoint, but it should be possible to get 200% text enlargement in some way.

This redefines/limits the scope of 1.4.4 (a non-normative document redefining the normative requirement). And it further goes against the full pages conformance requirement https://www.w3.org/TR/WCAG21/#cc2

> A full page includes each variation of the page that is automatically presented by the page for various screen sizes (e.g. variations in a responsive Web page). Each of these variations needs to conform (or needs to have a conforming alternate version) in order for the entire page to conform.

I know this has been a tricky subject in general (with previous discussions like https://github.com/w3c/wcag/issues/1671 and https://github.com/w3c/wcag/issues/704), but this seems a fairly egregious case of redefining things by the backdoor again (and not even from the understanding in 1.4.4, but from a completely separate understanding document) - at least on the face of it/how it can be interpreted.

I think I remember the context for this being "users may not be able to zoom to 200% before hitting another breakpoint, so technically at that original breakpoint they can't get to 200% since they now fell into the next breakpoint", but this nuance isn't there in the current wording. I would suggest this needs to be expanded - and also, more crucially, this needs to be presented in the understanding document for 1.4.4 as well, rather than expecting authors/auditors to find this information about 1.4.4 in a completely separate understanding document for 1.4.10

**Comment by detlevhfischer** [comment]
Yes, I guess best delete this paragraph and amend 1.4.4 the normative text instead: #1671

**Comment by patrickhlauke** [comment]
i suggest we first do the former, and then get consensus on how to change the normative wording of 1.4.4 (if we even can at this stage) / soften it somehow in the understanding at least (adding some "reasonableness" clause in the understanding along the lines of what we said above). probably two steps i'd say.

**Comment by patrickhlauke** [comment]
Coming in late, but made an initial stab at this here https://github.com/w3c/wcag/pull/2630
